charts/incubator/eco/values.yaml

```diff
@@ -92,7 +92,6 @@ persistence:
         main:
           mountPath: /serverdata/serverfiles/Configs/Network.eco
           subPath: Network.eco
-          readOnly: true
```

Comment:
Configmaps cannot be writeable. Regardless of this setting

Reply:
what would be the other way config the file, an init container? i dont see why its needs to be writeable but that was the issue i faced on the current version.

Reply:
The app complains that must be writeable?
maybe it stores info there?

Reply:
all config is set...not much else for it to do.